packages/lodestar-validator/src/api/LocalClock.ts

```diff
@@ -45,12 +50,13 @@ export class LocalClock implements IBeaconClock {
     return computeEpochAtSlot(this.config, this.currentSlot);
   }
 
-  private abort = (): void => {
-    clearTimeout(this.timeoutId);
-    if (this.signal) {
-      this.signal.removeEventListener("abort", this.abort);
+  private async start(): Promise<void> {
```

Comment:
https://github.com/ChainSafe/lodestar/issues/1196 :(

Reply:
This function may not even exist as you can run this code directly on the constructor. I can rename to `runInternalProcess` or something to differentiate the pattern